crates/bridge-exec/src/handler.rs

```diff
@@ -297,6 +297,9 @@ where
                 if let Ok(msg) = msg {
                     let payload = msg.payload();
                     let parsed_payload = borsh::from_slice::<Payload>(payload);
+                    let scope = msg.scope();
+                    let parsed_scope = borsh::from_slice::<Scope>(scope);
+                    debug!(msg = ?msg, payload = ?parsed_payload, scope = ?parsed_scope, "got message from the L2 Client");
```

Comment:
`?msg`

Really I would do

* payload -> raw_payload
* parsed_payload -> payload

And the same for the scope so that you can also avoid the extra `=` and make it simpler in the consumer.